crates/core/src/test_utils/deploy_helpers.rs

```diff
@@ -98,12 +98,29 @@ pub fn get_contract(filename: &str) -> CompactContractBytecode {
     let dot_sol = format!("{filename}.sol");
     let dot_json = format!("{filename}.json");
 
-    let foundry_default_out = load_config().out;
```

Comment:
can we keep the load config and use your method if there is not config? It makes more sense imo to use to config if you find one

Reply:
So, I think we should avoid this, because:
- we would then need more logic, i.e checking whether `FOUNDRY_OUTPUT_PATH` exists or not, there are scenarious where someone might have forget to provide this variable, but RPC will still take the builds from local buildl, this can create confusion, as they might not be suspecting that they have forgot to provide this varialbe. { TLDR; I think it will make it more automagical which we don't seem to need as of now, this is used in just one place, if complexity increases, might we can do this. }